test/config_onlyFakes_multiBand

```diff
@@ -3,7 +3,7 @@
 #The command to run would look like:
 #multiBand.py /path/to/data --rerun=rerun_old:rerun_new --id tract=tract1^tract2 filter=filter1^filter2 patch=patch1^patch2 --queue small --nodes 4 --procs 9 --job myJob --clobber-config -C config_onlyFakes_multiBand --mpiexec='-bind-to socket' --time 1000
 
-#import fakes.mergeOnlyFakes
```

Comment:
Again, do you need the lines that are commented out?

Reply:
It was intended for the tests directory to be removed, as they no longer work with lsst, in lieu of unit tests being written in the future